File: MasterLogicaProgramacion/EjerciciosJavascript/Ej17.js
function buzz(numero) {
    let lista = [];
    
    for (let i = 1; i <= numero; i++) {
        lista.push(i);
    }

    lista = lista.map((x) => {
        if (x % 5 == 0 && x % 3 == 0) {
            return "BuzzLightYear";
        }
        else if (x % 5 == 0) {
            return "Lightyear";
        }
        else if (x % 3 == 0) {
            return "Buzz";
        }

        return x;
    });

    return lista;
}

console.log(buzz(15));

File: Cursos/MasterLogicaProgramacion/EjerciciosPython/Ej17.py
def buzz(numero):
    lista = []
    
    for i in range(1, (numero + 1), 1):
        lista.append(i)

    lista = ["BuzzLightYear" if (x % 5 == 0 and x % 3 == 0) else "Lightyear" if x % 5 == 0 else "Buzz" if x % 3 == 0 else x for x in lista]

    return lista

print(buzz(15))